lib/index.js

```diff
@@ -249,16 +291,48 @@ module.exports = (secp256k1) => {
       }
     },
 
-    ecdsaSign (msg32, seckey, options = {}, output) {
+    ecdsaSign (msg32, seckey, options = {}, output, forceLegacy) {
       isUint8Array('message', msg32, 32)
       isUint8Array('private key', seckey, 32)
-      assert(toTypeString(options) === 'Object', 'Expected options to be an Object')
-      if (options.data !== undefined) isUint8Array('options.data', options.data)
-      if (options.noncefn !== undefined) assert(toTypeString(options.noncefn) === 'Function', 'Expected options.noncefn to be a Function')
+      assert(
+        toTypeString(options) === 'Object',
+        'Expected options to be an Object'
+      )
+      if (options.data !== undefined) {
+        isUint8Array('options.data', options.data)
+      }
+      if (options.noncefn !== undefined) {
+        assert(
+          toTypeString(options.noncefn) === 'Function',
+          'Expected options.noncefn to be a Function'
+        )
+      }
```

Comment:
just style changes? maybe we can undo to make it easier to review

Reply:
It's weird I used `standard` to fix it. I feel like it wasn't formatted before

Reply:
let's format it in a separate PR then, i don't want to confuse functional with style changes, they have very diff risk levels